src/intervals/special_intervals.jl

```diff
@@ -33,6 +33,7 @@ nai() = nai(get_interval_precision()[1])
 
 isnai(x::Interval) = isnan(x.lo) || isnan(x.hi)
 
+isfinite(x::Interval) = isfinite(x.lo) && isfinite(x.hi)
 
```

Comment:
Since you are including `isfinite` here, I think it would be worth to also include `isnan()` and `isinf()`, which are somehow missing in ValidatedNumerics (and are necessary to a nicer experience when combining this with TaylorSeries.jl). We already have something that implements this functionality, but these functions are more julian.
As far as I can tell, `isnan()` is equivalent to `isnai()`, and `isinf()` is equivalent to `isunbounded()` (or to the negation of `isfinite()`). Could you include them here?